hilti/runtime/include/types/time.h

```diff
@@ -36,14 +36,15 @@ class Time {
      * Constructs a time from a double value.
      *
      * @param secs seconds since the UNIX epoch.
+     * @throws OutOfRange if *secs* cannot be represented with the internal resolution
```

Comment:
The formatting of `secs` will be inconsistent with other fields. Let's either leave it plain (`secs` instead of `*secs*`) like in the line above, or format them all as code inside backticks.

Reply:
actually it's not: `*...*` is used more often than backticks. Not saying that that's the better way of doing it, but currently we're inconsistent either way.